CONTRIBUTING.md

```diff
@@ -32,9 +56,54 @@ Please take a look at the current open issues to see if any of them interest you
 5. Describe the problem and add references to the related issues that the request solves.
 6. Request review of your code by other contributors. Try to improve your code as much as possible to lessen the burden on others.
 
+## Merge Requests
```

Comment:
We have a "Pull Requests" subsection just above this.  I personally don't distinguish between "Pull" and "Merge" requests.  Is there a difference?  If not, we should combine these two subsections.  If so, maybe we need a 1-liner to explain the distinction?  (Note that below, in the paragraph after the list, the term "pull request" is used twice.  That leads me to believe this subsection should be combined with the previous one.)

Reply:
I realized now that pull requests were called merge requests before. I will merge the two.